src/openforms/registrations/contrib/zgw_apis/models.py

```diff
@@ -57,6 +58,20 @@ class ZgwConfig(SingletonModel):
         validators=[validate_rsin],
         help_text=_("Default RSIN of organization, which creates the ZAAK"),
     )
+    vertrouwelijkheidaanduiding = models.CharField(
+        _("vertrouwelijkheidaanduiding"),
+        max_length=24,
+        choices=VertrouwelijkheidsAanduidingen.choices,
+        default=VertrouwelijkheidsAanduidingen.zaakvertrouwelijk,
+        help_text=_(
+            "Indication of the level to which extend the dossier of the ZAAK is meant to be public."
```

Comment:
```suggestion
            "Indication of the level to which extend the dossier of the ZAAK is meant to be public."
```
is this actually used for the zaak or only the documents of a zaak?